src/LiquidCrystal.h

```diff
@@ -82,12 +82,17 @@ class LiquidCrystal : public Print {
   void setCursor(uint8_t, uint8_t); 
   virtual size_t write(uint8_t);
   void command(uint8_t);
+  uint8_t read_BF_addr(void);
+  uint8_t read_RAM(void);
```

Comment:
The other functions in the library use camelCase names. Is there a reason you departed from that naming convention for these functions?

Please add the function names added to the public API to [keywords.txt](https://github.com/arduino-libraries/LiquidCrystal/blob/master/keywords.txt), using a true tab as the field separator:
https://arduino.github.io/arduino-cli/library-specification/#keywordstxt-format

Reply:
Thanks for pointing out my mistakes. I have corrected the case and added function names to keywords.
Also I note that command function keyword was missing. Was there a reason or should I add it.

Reply:
>I note that command function keyword was missing. Was there a reason

Probably they just forgot to add it. As you probably noticed, it's an easy thing to miss since keywords are purely cosmetic.

>should I add it

I think it would be a good idea (but I'm not the person who decides which pull requests are merged). Please submit that as a separate pull request though, since it's unrelated to the change you propose here.

Reply:
I replied here earlier, but I think I accidentally deleted it, so I'll try again:

>I note that command function keyword was missing. Was there a reason

Probably they just forgot to add it. As you might have noticed, it's easy to do that since keywords are purely cosmetic.

>should I add it

I think it would be nice (though I'm not the one who decides which pull requests get merged). If you want to add it, please do so in a separate pull request, since that change is unrelated to the one you propose in this pull request.